### ISSUE TITLE
Black box appears when hiding captions/subtitles on Vudu

### ISSUE THREAD

**Issue by richardfrost** [richardfrost/AdvancedProfanityFilter#430]
#### :bug: Description
When watching a video on Vudu, there is a small black square that moves around the screen.

#### :pencil: Additional context
Requested by a user on the Chrome Web Store

**Comment by richardfrost** [comment]
I think this Custom Site config should fix the problem:

```json
{
  "www.vudu.com": [
    {
      "displaySelector": "div#subtitleContainer > div",
      "mode": "element",
      "subtitleSelector": "span.subtitles",
      "tagName": "DIV"
    }
  ]
}
```

If you get around to testing it let me know, but I'll plan on updating it on the next release.

**Comment by richardfrost** [comment]
This is included in [Version 2.37.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.37.0). If anyone has a chance to test this out, feel free to reply here to report back how it works for you. Thanks!
